contract/contracts/AstarBase.sol

```diff
@@ -91,4 +95,16 @@ contract AstarBase is Ownable {
     function pause(bool _state) public onlyOwner {
         paused = _state;
     }
+
+    /// @notice set new value for the unregisterFee
+    /// @param _newCost new fee cost
+    function setUnregisterFee(uint256 _newCost) public onlyOwner {
+        unregisterFee = _newCost;
+    }
+
+    /// @notice withdraw contract's funds
+    function withdraw() public payable {
+        (bool dev, ) = payable(owner()).call{value: address(this).balance}("");
```

Comment:
If I am getting this correctly, this function will pay out the unregister fee to the contract owner? (ignoring the fact that there is a fee for unregistering while register is free is strange from a semantic perspective).

I there a way to send the funds to the treasury or something if we want to keep this as a common-goods contract?

Reply:
If you give me treasury address I can use it.
Or faucet address?

Reply:
The faucet sounds like a great idea!
`0x91986602d9c0d8A4f5BFB5F39a7Aa2cD73Db73B7`

Reply:
@hoonsubin please check. Updated